Fix #1905, Unassigned staff error is shown in the same page

iamsk9 opened the pull request:
Now In productive collection sheet if staff is not assigned error message is displayed in 
same page.

Please view the changes made and tell me whether it is working or not.

Thank You :)

mbj36 commented on app/views/collection/collectionsheet.html:
```diff
@@ -6,15 +6,20 @@
 	<div>
+			<div ng-show="noStaffError">
+					<div class="alert alert-danger">
+							<strong>Staff is mandatory</strong>
```
It should not be hard-coded

mbj36 commented:
@iamsk9 I see there are two accounts, you are commenting from different accounts and making changes from different account

iamsk9 commented:
Sorry for the hard coding. The code is nicely indented in my editor but got some extra spaces here. I fixed it. I didn't see that my git config user.email has been changed. But i changed that now. My git email id is iamsk1996@gmail.com

nazeer1100126 commented:
This PR have conflicts, also check whether this issue is reproducible on latest code.
